docs/faq.md

```diff
@@ -2,53 +2,113 @@
+    - **Maps**: Maps are **required to run AzerothCore**. Maps provide the physical values and data for the core to interpret. Based on these, the core has a layout that can be compared with each client. This includes area definitions.
+    - **VMaps**: VMaps are **optional, but highly recommended**. VMaps ("Virtual Maps") calculate the possibility of line-of-sight, as an example. Based on their content, the server can (for example) calculate if spell casts are possible (e.g. if a wall is in between the target and caster or not).
+    - **MMaps**: MMaps are optional, but recommended. To further physical boundaries, MMaps ("Movement Maps") enforce physical boundaries on non-player characters, e.g. NPCs, as their collision is not handled by the client. They also improve path generation.
+    - **DBC**: DBCs are **required to run AzerothCore**. The DBCs ("Data Base Client [Files]") give essential values that the World of Warcraft client interprets. They define races, textures, local models and more. AzerothCore interprets these and loads them up.
+  - Compiling AzerothCore with "TOOLS" will always create the necessary tools to extract and create these files.
 
-## DEVELOPMENT QUESTIONS
+- I have issues with extracting Maps, VMaps, MMaps, DBCs (and I've tried using older versions of the extractors) - what is wrong?
+  - The older tools are deprecated and will not work.
+  - You are required to use a commandline to be able to run the tools.
 
-#### 1 - I need a new hook for my custom module, what can I do?
+- I am missing the libraries for MySQL and can't seem to find them in the repository?
+  - The library is called "mysql.lib" and is not provided by AzerothCore.
+  - Make sure you installed your MySQL-Server with DEVELOPMENT HEADERS.
+  - You can follow the [Core Installation](core-intallation.md) guide to find the libraries.
 
-You can add the hook to your own fork (cf: [Create a new Hook](http://www.azerothcore.org/wiki/Create-a-new-Hook)) and create a new Pull Request to the official repository so we can validate it and merge it.
+- I am missing the libraries for OpenSSL and can't seem to find them in the repository?
+  - For OpenSSL versions prior to 1.1.0 they will be named "libeay32.dll" and "ssleay32.dll".
+  - For OpenSSL versions 1.1.0 and more recent they are called:
+    - For 32bit installs: "libssl-1_1.dll" and "libcrypto-1_1.dll".
+    - for 64bit installs: "libssl-1_1-x64.dll" and "libcrypto-1_1-x64.dll"
+  - You can follow the [Core Installation](core-intallation.md) guide to find the libraries.
+
+## Debug-related FAQ
+
+- How can I get a good crashlog on Windows?'
+  - Compile your core in RelWithDebInfo or Debug. A crashlog from Release will be useless.
+
+## Module-related FAQ
+
+- I need a new hook for my custom module, what can I do?
+  - You can add the hook to your own fork (cf: [Create a new Hook](create-a-new-hook.md)) and create a new Pull Request to the official repository so we can validate it and merge it.
```

Comment:
it's also worth linking this: https://stackoverflow.com/questions/66340549/is-it-possible-to-turn-a-core-patch-into-a-module-for-azerothcore/66340683#66340683